EIPS/eip-1.md

```diff
@@ -103,9 +93,11 @@ Each EIP must begin with an RFC 822 style header preamble, preceded and followed
 
 ` * discussions-to:` <url>
 
-` status:` <Draft | Active | Accepted | Deferred | Rejected | Withdrawn | Final | Superseded>
+` status:` <Draft | Last Call | Approved | Final | Active | Deferred | Rejected | Superseded>
+
+`* review-period-end-date: YYYY-MM-DD
```

Comment:
Maybe `review-period-ends` for conciseness?

Reply:
Sure, updated in 46e54b2